doc/guides/bluetooth/gap-pics.rst

```diff
@@ -767,4 +805,5 @@ TSPC_GATT_1_2	True		GATT Server Role (O)
 TSPC_SM_1_1	True		Master Role (Initiator)
 TSPC_SM_1_2	True		Slave Role (Responder)
 TSPC_SM_2_4	False (*)	OOB supported (O)
+TSPC_LL_2_5	False (*)	Resolution of private addresses (O)
```

Comment:
Isn't the above still under the "Peripheral Simultaneous BR/EDR and LE Transports" section? If so then I suppose everything should be False since we don't do BR/EDR?

Reply:
"Peripheral Simultaneous BR/EDR and LE Transports" are GAP_45_1 and GAP_45_2, which are False.
SM_1_1 and SM_1_2 are dependencies to GAP/IDLE/NBON/BV-01,02,03-C.